public/app/plugins/panel/heatmap/panelcfg.cue

```diff
@@ -14,19 +14,101 @@
 
 package grafanaplugin
 
+import (
+	ui "github.com/grafana/grafana/packages/grafana-schema/src/common"
+)
+
 composableKinds: PanelCfg: {
+	// maturity: "experimental"
 	lineage: {
 		seqs: [
 			{
 				schemas: [
 					{
+						HeatmapColorMode:  "opacity" | "scheme"     @cuetsy(kind="enum")
+						HeatmapColorScale: "linear" | "exponential" @cuetsy(kind="enum")
+						HeatmapColorOptions: {
+							mode:     HeatmapColorMode
+							scheme:   string            // when in scheme mode -- the d3 scheme name
+							fill:     string            // when opacity mode, the target color
+							scale:    HeatmapColorScale // for opacity mode
+							exponent: float32           // when scale== sqrt
```

Comment:
I think the comments need to be on top of the line and not next to it, otherwise they won't be picked up by our comment generation tools. 
```suggestion
                                                         // when scale== sqrt
							exponent: float32           
```

Reply:
That's true. Those were remnants of the old manual models.gen file. I updated the docs better overall